cookbook/case_studies/ml_training/house_price_prediction/house_price_predictor.py

```diff
@@ -116,7 +117,9 @@ def gen_houses(num_houses) -> pd.DataFrame:
 # Data Preprocessing and Splitting
 # ===================================
 #
-# We split the data into train, test, and validation subsets.
+# We create two columns from the dataset- `feature` and `target`. 
+#
+# Next, we split these columns into train and test subsets. We also split the input DataFrame into train, validation, and test datasets.
```

Comment:
Let's keep this simple. Can you please revert? We need not really dive into how the data split is done.